api/README.md

```diff
@@ -0,0 +1,39 @@
+Authentication and API
+======================
+
+
+Authentication
+--------------
+
+CMS API require [OpenID Connect 1.0](https://openid.net/connect/) for 
+authentication.
+
+
+Authorization
+-------------
+
+A client application who need to fetch CMS data for a particular user need
+to ask some specific OAuth2 scopes. Exemples of CMS scopes:
+
+- `urn:cms:personal-informations:read`: read access to the user's basic 
+  personal informations
+- `urn:cms:civil-status:read`: read access to the user's civil status
```

Comment:
Why not "civil-informations" ? The term "status" suggests that there will be changes